Firebase/Auth/Source/Public/FIRAuth.h

```diff
@@ -534,10 +566,10 @@ NS_SWIFT_NAME(Auth)
     @remarks See `FIRAuthErrors` for a list of error codes that are common to all API methods.
  */
 - (void)signInWithCustomToken:(NSString *)token
-                   completion:(nullable FIRAuthResultCallback)completion;
+                   completion:(nullable FIRAuthDataResultCallback)completion;
 
 /** @fn signInAndRetrieveDataWithCustomToken:completion:
-    @brief Asynchronously signs in to Firebase with the given Auth token.
+    @brief Please use `signInWithCustomToken:completion:` instead.
```

Comment:
Are these deprecation messages friendly enough for Swift users?

Reply:
Added swift friendly deprecation warnings.